src/main/java/org/cactoos/list/Immutable.java

```diff
@@ -55,10 +66,49 @@
 
     /**
      * Ctor.
-     * @param src Source
+     * @param items Source array
+     */
+    @SafeVarargs
+    public Immutable(final T... items) {
+        this(new IterableOf<>(items));
+    }
+
+    /**
+     * Ctor.
+     * @param src Source collection
+     */
+    public Immutable(final Collection<T> src) {
+        this(new IterableOf<>(src.iterator()));
```

Comment:
@iakunin change the code for something like this:
```
super(() -> {
    final List<T> copy = new ArrayList<>(src.size());
    copy.addAll(src);
    return copy;
});
```

Reply:
@fabriciofx fixed